config/experiments/tree.yaml

```diff
@@ -0,0 +1,56 @@
+# @package _global_
+
+defaults:
+   - override /env: tree
+   - override /gflownet: trajectorybalance
+   - override /proxy: tree
+   - override /logger: wandb
+
+# Environment
+env:
+  max_depth: 5
+  reward_func: boltzmann
+  reward_beta: 8
+
+# GFlowNet hyperparameters
+gflownet:
+  random_action_prob: 0.1
+  optimizer:
+    batch_size:
+      forward: 100
+      backward_dataset: 0
+      backward_replay: 0
+    lr: 0.00001
+    z_dim: 16
+    lr_z_mult: 1000
+    n_train_steps: 40000
```

Comment:
40000 steps by default may be too much?

Reply:
Yeah, I wasn't sure how quick convergence to expect / how appropriate lowering the number of steps would be without tuning the remaining parameters (these are taken from alanine dipeptide experiment), so I just started with a long training and figured I would lower it later. But feel free to propose any other value.